tool/generate_analysis.dart

```diff
@@ -1103,8 +1107,7 @@ final String _staticFactory = r'''
     if (clientVersion != null) args.add('--client-version=$clientVersion');
 
     Process process = await Process.start(vmPath, args, environment: processEnvironment);
-    // ignore: unused_local_variable
-    Future unawaited = process.exitCode.then((code) => processCompleter.complete(code));
+    unawaited(process.exitCode.then((code) => processCompleter.complete(code)));
```

Comment:
Can you confirm that this matches the current min. sdk in the pubspec (`2.15.0`)? I don't remember when `unawaited` was added.

Reply:
Per https://github.com/dart-lang/sdk/blob/master/sdk/lib/async/future.dart#L954 `unawaited` has `@Since("2.15")` on it.